maps/submaps/surface_submaps/wilderness/demonpool.dmm

```diff
@@ -440,7 +440,6 @@
 	},
 /area/submap/DemonPool)
 "IP" = (
-/obj/effect/spawner/lootdrop,
```

Comment:
Er, is this just outright removing the loot from this POI?

Reply:
I was worried about that, but looking at the code, I don't actually think those particular spawners dropped any loot. it's been a hot minute since I booted up and compared (since this pr has been open a while) but I seem to recall that when I compared the before and after, the stuff the PoI spawns was handled in a different way. that being said, I can make some time to check again if you like

Reply:
Ah, yeah that code was dependent upon mapping to set the loot list.